src/theme.ts

```diff
@@ -0,0 +1,187 @@
+import { css } from 'styled-components';
+
+const palette = {
```

Comment:
why rgb rather than hex?

Reply:
Leaving the values rgb from the tutor sass side was helping me verify I was moving them over correctly, I can probably convert them now though, I also prefer hex.